src/System.Reflection.Context/src/System.Reflection.Context.csproj

```diff
@@ -12,7 +12,6 @@
          but do not include this assembly in their framework.
```

Comment:
I was going to say no. but then I just had a look and it did get sucked into netstandard.dll as it lives in mscorlib.